openmc/geometry.py

```diff
@@ -110,53 +105,43 @@ def find(self, point):
         """
         return self.root_universe.find(point)
 
-    def get_cell_instance(self, path):
-        """Return the instance number for the final cell in a geometry path.
+    def get_instance(self, path):
+        """Return the instance number for a cell/material in a geometry path.
```

Comment:
You might note that this returns the indices starting from zero to index into the Python array, not starting from 1 as is done for the Fortran array.